examples/complete/README.md

```diff
@@ -1,3 +1,10 @@
+#################################################################################################################
```

Comment:
This may be difficult to maintain. It'll be tough to revert these changes once we shift the modules to only support terraform v1.0.0+

My preference would be to create a new example with this contents, and we can delete the example as soon as we stop supporting older versions of terraform.

Remind me, the issue in the security groups module was a terraform bug that impacted count, right?

Could we modify the security groups module to use for each instead?

Reply:
The idea here was to provide a solution for the examples. Our next step is to release a major version without these changes. Changing the security module requires more work, we did some testing and it doesn’t work just by replacing the “count” for the “for each” argument. Since this will be the last version of this major one we don’t think it is worth it. This version is for customers that have to run version 13, otherwise, we recommend changing to newer releases.

Reply:
This PR touches 33 files because we added the "warning" label but if we only remove the way we are creating the security groups we are only going to touch around 9 files, plus the README and CHANGELOG files.
If you find it more useful for your customers I can keep the examples with the "count" argument and create a directory for those using the tf version 0.13.7 and remove this in the next major upgrade.